pymc3/distributions/continuous.py

```diff
@@ -1246,23 +1245,19 @@ def logcdf(value, alpha, beta):
 
         Parameters
         ----------
-        value: numeric
-            Value(s) for which log CDF is calculated.
+        value: numeric or np.ndarray or aesara.tensor
+            Value(s) for which log CDF is calculated. If the log CDF for multiple
+            values are desired the values must be provided in a numpy array or Aesara tensor.
```

Comment:
Might as well keep this parameter description simple and put all the type-based information in type annotations.

Reply:
Yeah, these docstrings could benefit from an overhaul (or be removed altogether). I simply changed them to be equal to those of the remaining distributions.

Reply:
Opened an issue here: https://github.com/pymc-devs/pymc3/issues/4859